Toolset/palettes/behaviors/revcorestackbehavior.livecodescript

```diff
@@ -0,0 +1,815 @@
+      end if
+   end repeat
+   return tTrimmedText
+end ideCoreTrim
+
+function ideColorGet pTag
+   switch pTag
+      case "state_disabled"
+         return "208,208,208"
+         break
+      case "palette_background"
+      case "toolbar_background"
+         return "236,236,236"
+         break
+      case "palette_background_muted"
+      case "state_normal_foreground"
+      case "state_hover_foreground"
+      case "state_pressed_foreground"
+      case "state_selected_foreground"
+      case "state_disabled_foreground"
+      case "state_booleanTrue_foreground"
+      case "state_booleanFalse_foreground"
+      case "border_muted_hilite"
+      case "pb_line_border_left_hilite"
+      case "pb_script_lines_text"
+      case "pb_script_lines_text_hover"
+      case "pb_behavior_script_lines_text"
+      case "pb_behavior_script_lines_text_hover"
+      case "pb_edit_field_bg"
+      case "border_hilite"
+      case "pb_line_background"
+         return "255,255,255"
+         break
+      case "state_normal"
+      case "pb_disclosure_arrow"
+         return "129,129,129"
+         break
+      case "state_hover"
+      case "hilite_above"
+      case "pb_script_lines_hover"
+      case "pb_behavior_script_lines_hover"
+      case "pb_drag_hilite_top"
+         return "0,72,204"
+         break
+      case "state_pressed"
+      case "hilite_below"
+      case "pb_drag_hilite_bottom"
+         return "192,0,204"
+         break
+      case "state_selected"
+      case "pb_script_lines"
+      case "pb_behavior_script_lines"
+         return "0,195,255"
+         break
+      case "pb_no_script_lines"
+         return "157,233,255"
+         break
+      case "state_booleanTrue"
+         return "100,100,100"
+         break
+      case "state_booleanFalse"
+         return "182,182,182"
+         break
+      case "state_muted_pressed"
+         return "231,22,255"
+         break
+      case "state_muted_selected"
+      case "pb_line_selected"
+         return revEnvironmentEditionProperty("pb_line_selected_color")
+         break
+      case "state_muted_disabled"
+      case "hilite_center"
+      case "pb_drag_hilite_center"
+      case "pb_line_selected_hover"
+         return "255,211,211"
+         break
+      case "state_muted_selected_secondary"
+      case "pb_line_selected_trail"
+         return "244,244,244"
+         break         
+      case "border"
+      case "pb_list_bottom_border"
+      case "pb_edit_field_border"
+      case "pb_line_border_left"
+         return "225,225,225"
+         break
+      case "border_muted"
+      case "pb_line_border_bottom"
+         return "234,234,234"
+         break        
+      case "text"
+         return "0,0,0"
+         break
+      case "text_muted"
+         return "95,95,95"
+         break
+      case "pb_palette_background"
+      case "pb_line_show_select_background"
+         return "236,236,236"
+         break
+   end switch
+end ideColorGet
```

Comment:
Presume this can be ditched for the revEnvironment API

Reply:
I'm not really sure what you mean here

Reply:
Oh... do you mean the ide lib API?

Reply:
I meant `revEnvironmentEditionProperty` but maybe not all these things are implemented there

Reply:
Ah, these are not edition specific :-D sorry, being an idiot

Reply:
yeah I've only moved stuff that was conditional on edition